vllm/config.py

```diff
@@ -358,6 +403,12 @@ def verify_with_parallel_config(
                            "fallback to the eager mode.")
             self.enforce_eager = True
 
+        if pipeline_parallel_size > 1 and self.use_async_output_proc:
+            raise NotImplementedError(
+                "Async output processor is not supported with "
+                "pipeline parallelism currently. Disable it"
+                " with --disable-async-output-proc.")
```

Comment:
Why don't we do `self.use_aysnc_output_proc = False` instead?

Reply:
Since async output proc is enabled by default, it made more sense to use to add an optional flag to disable. `--disable-async-output-proc`.

Reply:
fixed